modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/sxp_blueprint_admin/js/shared/CodeMirrorEditor.js

```diff
@@ -441,6 +538,31 @@ function isObjectProperty(token) {
 	);
 }
 
+/**
+ * Removes any duplicate properties in an array with the same type and name.
+ * This could happen in some cases like when flattening properties from `anyOf`
+ * in a schema.
+ * @param {Array} items An array of properties
+ * @returns {Array}
+ */
+function removeDuplicateProperties(items) {
```

Comment:
Instead of naming the parameter as `items` we should be more specific like `properties` since this function specifically works with a certain object type and not all general "items". 

We should also specify the expected type of object in the array. For example: 
```
A property should have a `name` and `type`.
```

Reply:
Thanks for the feedback! Specified properties and added that description in 👍